conversant/demo/streamlit_example.py

```diff
@@ -128,6 +127,26 @@ def update_prompt_from_json() -> None:
                 0, f"from launch_demo: {st.session_state.bot.persona_name}"
             )
 
+    # Adding a header to direct users to sign up for Cohere, explore the playground,
+    # and check out our git repo.
+    st.header("🎭 Conversational personas using Cohere")
```

Comment:
```suggestion
    st.header("🎭 Conversational Personas using Cohere")
```
nit: capitalization of "Personas"